mode-apis-base/src/alertingModels.ts

```diff
@@ -163,12 +163,47 @@ export const isMetricsThresholdAlertCondition = (obj: unknown): obj is MetricsTh
 
 
 
-/**
- * TODO - Define the attributes required for AlertAction
- */
-export interface AlertAction {
+export interface AlertMessageTemplate {
+    readonly title: string;
+    readonly message: string;
+    readonly recoveryMessage?: string;
+}
+
+export type AlertRecipientType = 'email';
+
+export interface AlertRecipient {
+    recipientType: AlertRecipientType;
 }
 
+export interface AlertRecipientEmail extends AlertRecipient {
+    readonly recipientType: 'email';
+    readonly email: string;
+}
+
+// We may want to have more various recipient types in the future.
```

Comment:
This is just for showing some more examples of Recipients. 
I will remove these comment-out lines after the review.